examples/documentation.html

```diff
@@ -327,7 +327,7 @@
 
         <div style="height: 50px"></div>
 
-		<h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>
+    <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>

```

Comment:
I don't follow you when you say _...it rather adds a useful api information to the documentation..._ unless you meant that it toggles the _display of the api information_

but taking a look again at [your change](https://github.com/dmapper/nvd3/blob/master/examples/documentation.html#L330), I ask myself if I am totally blind and missed something that made you want to change this file more than "fixing the indentation". Because as it looks to me right now in your change of this file, it didn't make the indentation any better even if the [previous line in master branch](https://github.com/novus/nvd3/blob/master/examples/documentation.html#L330) *obviously looks incorrect*, as you say.